### PR TITLE
Restructured example apps to have cleaner HDFS installation/sys tests

### PR DESCRIPTION
**Pull request by potiuk**

### CODE REVIEW

File: oozie-to-airflow/README.md [hunk 1]
```diff
@@ -388,6 +388,20 @@ Cloud Dataproc Cluster with Oozie
 * Hadoop version
 * Init action: TODO
 
+### Example apps
+
+This folder contains example applications that can be run to run both Composer and Ooozie jobs for the apps
```
Comment (sprzedwojski) [inline oozie-to-airflow/README.md]:
```suggestion
This folder contains example applications that can be run to run both Composer and Oozie jobs for the apps
```

File: oozie-to-airflow/README.md [hunk 2]
```diff
@@ -388,6 +388,20 @@ Cloud Dataproc Cluster with Oozie
...
+
+Structure of the application folder:
+
+```
+<APPLICATION>/
+             |- workflow.xml    - Oozie workflow xml (1.0 schema)
+             |- job.properties  - job properties that are used to run the job
+             |- assets          - folder with extra assets that should be copied to application folder in HDFS
+             |- configuration.template.properties - template of configuration values used during conversion
+             |- configuration.properties - generated properites for configuration values
```
Comment (sprzedwojski) [inline oozie-to-airflow/README.md]:
```suggestion
             |- configuration.properties - generated properties for configuration values
```

File: oozie-to-airflow/README.md [hunk 3]
```diff
@@ -436,24 +449,52 @@ Flags:
...
+    .COMPOSER_NAME
+    .DATAPROC_CLUSTER_NAME
+    .GCP_REGION
+    .LOCAL_APP_NAME
+    .PHASE
+
+### Phases of the tests
+The following phases are defined for the system tests:
+
+* prepare-configuration - prepares configuration based on passed dataproc/composer parameters
```
Comment (sprzedwojski) [inline oozie-to-airflow/README.md]:
```suggestion
* prepare-configuration - prepares configuration based on passed Dataproc/Composer parameters
```

File: oozie-to-airflow/README.md [hunk 4]
```diff
@@ -436,24 +449,52 @@ Flags:
...
+* convert - converts the example application workflow to DAG and stores it in output/<APPLICATION> directory
+* prepare-dataproc - prepares dataproc cluster to execute both Composer and Oozie jobs. The preparation is:
```
Comment (sprzedwojski) [inline oozie-to-airflow/README.md]:
```suggestion
* prepare-dataproc - prepares Dataproc cluster to execute both Composer and Oozie jobs. The preparation is:
```

**Comment by potiuk** [comment]
Merging

**Pull request merged**